src/main/java/frc/robot/RobotContainer.java

```diff
@@ -96,69 +100,57 @@ public RobotContainer() {
+    colorSensor = Constants.HAVE_COLOR_SENSOR ? new ColorSensor() : null;
+    feeder = Constants.HAVE_FEEDER ? new Feeder() : null;
+    flywheel = Constants.HAVE_FLYWHEEL ? new Flywheel() : null;
+    intake = Constants.HAVE_INTAKE ? new Intake() : null;
+    leds = Constants.HAVE_LEDS ? new Leds() : null;
 
     // ========================= Autonomous =========================
     // ---------- Create Named Commands for use by Path Planner ----------
     NamedCommands.registerCommand("Spin 180", DriveCommands.spinCommand(driveBase, Rotation2d.fromDegrees(180), 1));
-    NamedCommands.registerCommand("StartIntake", LightsCommands.blinkCommand(lightsSubsystem, Color.kPurple));
+    if (Constants.HAVE_LEDS) {
+      NamedCommands.registerCommand("StartIntake", LightsCommands.blinkCommand(leds, Color.kPurple));
+    }
 
     Command aimCommand = new ConditionalCommand(
         // Turn to Blue Speaker.
         DriveCommands.turnToTargetCommand(driveBase,
-            new Translation2d(Units.inchesToMeters(-1.5), Units.inchesToMeters(218.42)), 4.5),
+            Constants.BLUE_SPEAKER_LOCATION, 4.5),
         // Turn to Red Speaker.
         DriveCommands.turnToTargetCommand(driveBase,
-            new Translation2d(Units.inchesToMeters(652.73), Units.inchesToMeters(218.42)), 4.5),
+            Constants.RED_SPEAKER_LOCATION, 4.5),
         () -> DriverStation.getAlliance().get() == DriverStation.Alliance.Blue);
     Command autoShootCommand;
     if (Constants.HAVE_SHOOTER) {
-      autoShootCommand = ShooterCommands.shootCommand(flywheel, feeder, lightsSubsystem, sensor);
-    } else {
-      autoShootCommand = LightsCommands.blinkCommand(lightsSubsystem, Color.kOrange);
+      autoShootCommand = ShooterCommands.shootCommand(flywheel, feeder, leds, colorSensor);
+    } else if (Constants.HAVE_LEDS) {
+      autoShootCommand = LightsCommands.blinkCommand(leds, Color.kOrange);
+    }
+    if (Constants.HAVE_SHOOTER || Constants.HAVE_LEDS) {
+      NamedCommands.registerCommand("ShootNote", new SequentialCommandGroup(aimCommand, autoShootCommand));
     }
-    NamedCommands.registerCommand("ShootNote", new SequentialCommandGroup(aimCommand, autoShootCommand));
 
     // ---------- Set-up Autonomous Choices ----------
     autoChooser = new LoggedDashboardChooser<>("Auto Choices", AutoBuilder.buildAutoChooser());
 
     // ========================= Tele-Op =========================
     // ---------- Configure Joystick for Tele-Op ----------
-    driveBase.setDefaultCommand(DriveCommands.joystickDrive(driveBase,
+    driveBase.setDefaultCommand(DriveCommands.manualDriveDefaultCommand(driveBase,
         () -> -controller.getLeftY(),
         () -> -controller.getLeftX(),
-        () -> -controller.getLeftTriggerAxis() + controller.getRightTriggerAxis()));
+        () -> controller.getLeftTriggerAxis() > controller.getRightTriggerAxis()
+            ? controller.getLeftTriggerAxis()
+            : -controller.getRightTriggerAxis()));
```

Comment:
Is the sign change here intentional?

Reply:
Yes, the triggers were backwards the other way.